.github/workflows/codeql-daily.yml

```diff
@@ -1,44 +0,0 @@
-name: CodeQL (daily)
-
-on:
-  schedule:
-    # Daily at 01:30 (UTC)
-    - cron: '30 1 * * *'
-  workflow_dispatch:
-
-jobs:
-  analyze:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up Java 17
-        uses: actions/setup-java@v4
-        with:
-          distribution: temurin
-          java-version: 17
-
-      - name: Initialize CodeQL
-        uses: github/codeql-action/init@v3
-        with:
-          languages: java
-          # using "latest" helps to keep up with the latest Kotlin support
-          # see https://github.com/github/codeql-action/issues/1555#issuecomment-1452228433
-          tools: latest
-
-      - name: Set up gradle
-        uses: gradle/actions/setup-gradle@v4
-      - name: Assemble
-        # skipping build cache is needed so that all modules will be analyzed
-        run: ./gradlew assemble --no-build-cache
-
-      - name: Perform CodeQL analysis
-        uses: github/codeql-action/analyze@v3
-
-  open-issue-on-failure:
```

Comment:
We loose this. Will passing codeql become part of the required status checks for a build to pass?

Reply:
yes

Reply:
(not quite yet though, I'll send an update to `repository-settings.md` later, I didn't want to add the required check right away since it will require all PRs to rebase to pass)